mlir/utils/performance/perfRunner.py

```diff
@@ -793,6 +819,133 @@ def benchmarkMIOpenWithMLIRKernels(configs, arch, filename, paths: Paths):
     chip = GFX_CHIP_RE.search(arch).group(0)
     df.to_csv(chip + '_' + filename, index=False)
 
+def findRunCommand(filename, includes=['RUN']):
+    with open(filename, 'r') as f:
+        for line in f:
+            if line.strip().startswith("//") and not 'DISABLE' in line and all(word in line for word in includes):
+                command = line.split("RUN")[1].strip()[2:] # Remove the "//" and "RUN" prefixes and leading/trailing whitespace
+                parts = command.split('|')  # Split the command using the "|" separator
+                return parts[0].strip()
+                # Stop processing lines after finding the first matching line
+    # Not a valid test
+    print("WARNING: cannot find valid RUN command in ", filename)
+    return ""
+
+def getTestVector(file_path, paths:Paths):
+    firstCommand = findRunCommand(file_path)
+    if (not firstCommand):
```

Comment:
Redundant parentheses. Also, maybe `firstCommand is None` given above suggestions.